modin/pandas/series.py

```diff
@@ -1376,14 +1341,28 @@ def nlargest(self, n=5, keep="first"):  # noqa: PR01, RT01, D200
         """
         Return the largest `n` elements.
         """
-        return self._default_to_pandas(pandas.Series.nlargest, n=n, keep=keep)
+        if len(self._query_compiler.columns) == 0:
+            raise NotImplementedError(
+                "Series.nlargest is not implemented for empty Series."
```

Comment:
Is this true for pandas as well?

Reply:
Hmm, just checked, and in pandas it returns `Series([], dtype: float64)` thingy...